library/tests/test_genre.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+from library.models import Genre
+
+
+class GenreTestCase(TestCase):
+    def test_cant_add_two_genres_with_same_name(self):
```

Comment:
Możnaby sprawdzać czy ten sam gatunek przechodzi, jezeli pierwsza litera jest mała
np Horror horror

Reply:
Dobry pomysł dodam to do testu